src/index.css

```diff
@@ -104,12 +104,12 @@ body {
 }
 
```

Comment:
- [ ] Your design should be similar to the Zeplin design, Kindly use the same fonts and colors for all the elements,
   - You can add the separator between the Complet section and the Current chapter,
   - Also you can consider changing the profile color to blue so it's similar to the Zeplin design
   - The form element have too much spacing, kindly consider removing the space so your design is identical to the Zeplin one

|Your design|Zeplin design|
|-|-|
|![Screenshot_20220617_155306](https://user-images.githubusercontent.com/25227022/174323264-006f9baa-d86f-49be-93e0-337462789d80.png)|![Screenshot_20220617_155852](https://user-images.githubusercontent.com/25227022/174323521-b34b18cf-8ed5-4bb5-a00b-2da502c5a397.png)|